forge-gui/res/cardsfolder/upcoming/the_drum_mining_facility.txt

```diff
@@ -0,0 +1,8 @@
+Name:The Drum, Mining Facility
+ManaCost:no cost
+Types:Plane Earth
+T:Mode$ PlanarDice | TriggerZones$ Command | Execute$ RolledDie | TriggerDescription$ Whenever you roll the planar die, creatures you control get +1/+1 and gain haste until end of turn.

```

Comment:
ValidPlayer missing

Reply:
I followed the example of Ten Wizards Mountain and Stairs to Infinity.

Reply:
it's somewhat future-proofing because they might print a card that let's you roll during another players turn
currently _Fractured Powerstone_ exists already, but its timing restriction can't be bypassed